torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -112,6 +112,25 @@ int64_t SizeEq::getDynamicValue() const {
 
 std::string SizeEq::ToString() const { return "SizeEq"; }
 
+SizeNe::SizeNe(torch::lazy::Value a, torch::lazy::Value b)
+    : XlaNode(torch::lazy::OpKind{c10::Symbol::fromQualString("aten::ne")},
+              {a, b}, xla::ShapeUtil::MakeShape(xla::S64, {}), 1) {
```

Comment:
let' use `GetShapeDimensionType`, @vanbasten23 when can we update all those s64 in the sizeNode to `GetShapeDimensionType`?

Reply:
Ack. I'm blocked on https://github.com/pytorch/xla/pull/4338 to update all return type: I could've done the casting to unblock me but I'd really like to understand where the s32/s64 mismatch comes from before blindly do the casting. I left a [comment](https://github.com/pytorch/xla/pull/4258#issuecomment-1355800759). Can you take another look @JackCaoG ?

On another thought, I'm not technically blocked by the above PR to update all those s64 in the sizeNode to use `GetShapeDimensionType `, let me do that today. Thanks for the reminder.